schemas/theme/theme_block.json

```diff
@@ -119,5 +104,65 @@
       "description": "When Shopify renders a block, it's wrapped in an HTML element with the shopify-block class. You can append other classes by using the class attribute.",
       "markdownDescription": "When Shopify renders a block, it's wrapped in an HTML element with the `shopify-block` class. You can append other classes by using the class attribute.\n\n---\n\n[Shopify reference](https://shopify.dev/docs/themes/architecture/blocks/theme-blocks/schema#class)"
     }
+  },
+  "definitions": {
+    "blocks_array": {
+      "type": "array",
+      "description": "A list of child blocks that you might want to include.",
+      "markdownDescription": "A list of child blocks that you might want to include.\n\n---\n\n[Shopify reference](https://shopify.dev/docs/themes/architecture/blocks/theme-blocks/schema#presets)",
+      "required": ["type"],
+      "items": {
+        "type": "object",
+        "properties": {
+          "$ref": "#/definitions/base_block",
+          "id": {
+            "type": "string"
+          }
+        },
+        "allOf": [
+          {
+            "if": {
+              "properties": {
+                "static": {
+                  "const": true
+                }
+              },
+              "required": ["static"]
+            },
+            "then": {
+              "required": ["id"]
+            }
+          }
+        ]
```

Comment:
The section schema already has this. It seemed an omission that the blocks didn't have anything about static blocks.